R/linear-mixed-model-development.R

```diff
@@ -29,6 +28,38 @@
 #' F leads to removal of rows containing NULLs.
 #' @param debug Provides the user extended output to the console, in order
 #' to monitor the calculations throughout. Use T or F.
+#' @param cores Number of cores you'd like to use. Defaults to 2.
```

Comment:
example shows 1 core and it might be good to stick with the default or not use cores in the example.